options.html

```diff
@@ -133,6 +134,13 @@ <h4>Version Choice</h4>
 							<input id="kk-version-both" name="kk-version" type="radio"><label for="kk-version-both"><span><span></span></span>Both</label>
 						</div>
 					</section>
+					<section>
+						<h4>Select songs</h4>
+						<div class="checkbox">
+							<input id="kk-songs-selection-enable" type="checkbox"><label for="kk-songs-selection-enable"><span></span>Only play the selected K.K. songs</label>
+						</div>
+						<select id="kk-songs-selection" style="height: 25vh" multiple></select>
```

Comment:
Maybe add some sort of indication to use the CTRL key to select multiple songs?

Reply:
I added a simple hint text